### PR TITLE
Extract Tencent Cloud 

### PR DESCRIPTION
**Pull request by sylviamoss**
Plugin's new home: https://github.com/hashicorp/packer-plugin-tencentcloud
Docs https://packer-git-extracttencentcloud-hashicorp.vercel.app/docs/builders/tencentcloud

### CODE REVIEW

File: website/content/docs/builders/vagrant.mdx [hunk 1]
```diff
@@ -1,217 +0,0 @@
----
```
Comment (sylviamoss) [inline website/content/docs/builders/vagrant.mdx:1]: Removing vagrant docs

File: website/content/docs/builders/cloudstack.mdx [hunk 2]
```diff
@@ -1,235 +0,0 @@
----
```
Comment (sylviamoss) [inline website/content/docs/builders/cloudstack.mdx:1]: Removing cloudstack docs

Comment (sylviamoss) [reply website/content/docs/builders/cloudstack.mdx:1]: https://packer-git-extracttencentcloud-hashicorp.vercel.app/docs/builders/cloudstack

File: website/content/docs/builders/vagrant.mdx (hunk 1 above)
Comment (sylviamoss) [reply website/content/docs/builders/vagrant.mdx:1]: https://packer-git-extracttencentcloud-hashicorp.vercel.app/docs/builders/vagrant

**Comment by azr** [comment]
:shipit:

**Pull request merged**